src/components/Title/index.js

```diff
@@ -0,0 +1,46 @@
+import { Cir1, Cir2, Cir3, Etc, DivTitle } from './style';
+import { useState } from 'react';
+
+const Index = () => {
+  const useConfirm = (message, deleteConfirm, cancelConfirm) => {
+    if (!deleteConfirm || typeof deleteConfirm !== 'function') {
+      return;
+    }
+    if (cancelConfirm && typeof cancelConfirm !== 'function') {
+      return;
+    }
+
+    const confirmAction = () => {
+      if (window.confirm(message)) {
+        deleteConfirm();
+      } else {
+        cancelConfirm();
+      }
+    };
+
+    return confirmAction;
+  };
```

Comment:
`deleteConfirm()`과 `cancelConfirm()` 함수가 그다지 길지 않기 때문에 따로 분리하여 이 함수에 props로 넘겨 주는 방법보다는 그냥 바로 이 안에 작성하셔도 괜찮을 것 같습니다!

```suggestion
const Index = () => {
  const useConfirm = () => {
    if (window.confirm('Tab을 닫을까요?')) {
      window.close();
    } else {
      return null;
    }
  };

  return (
    <DivTitle>
      <div style={{ display: 'flex' }}>
        <Cir1 onClick={useConfirm}>X</Cir1>
        <Cir2 />
        <Cir3 />
      </div>
      <Etc>⌥⌘1</Etc>
    </DivTitle>
  );
};
```